### PR TITLE
Adding lesson 7 on arrays.

### PR DESCRIPTION
**Pull request by owenlow**
Removing examples with objects, splitting up the exercises. Adding bookshelf analogy, slide on indexOf.

### CODE REVIEW

File: lesson7.md [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+let books = ["Harry Potter", "The Hobbit", "Game of Thrones"];
+```
+
+---
+
+### Accessing elements
+
+We can access elements in the array by number using square brackets "`[]`"
+
+The numbering starts at `0`:
```
Comment (sevtapim) [inline lesson7.md]: When talking about accessing elements using numbers, I would use a building analogy. Like the array is a building consisting of apartments and on each floor there is someone living on that floor. The building has ground floor, the first floor, etc. When we say array[0] we are basically asking who is living on the ground floor

Comment (owenlow) [reply lesson7.md]: Yes, I was thinking the same thing! Was planning to mention it verbally but I will also put a note in this slide

File: lesson7.md [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+Array is a container type that holds multiple values:
+
+```js
+let cars = ["Saab", "Volvo", "BMW"];
+
+// we create an array using []
+let emptyArray = [];
+
+// we put the values we want in these angled brackets
```
Comment (haraldF) [inline lesson7.md]: Isn't it `square brackets` (https://grammar.collinsdictionary.com/easy-learning/square-brackets) ? Angled seems to be `<>`: https://en.wiktionary.org/wiki/angle_bracket

Comment (owenlow) [reply lesson7.md]: Yes you're right, I think this was copied from the old slide

Comment (owenlow) [reply lesson7.md]: Fixed

**Pull request merged**